src/main/java/Duke.java

```diff
@@ -1,10 +1,134 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static Task[] taskList = new Task [100];
+    static int taskIndex=0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        greetUser("hi");
+        startChatbot();
+        greetUser("bye");
+    }
+
+    private static void startChatbot() {
+        Scanner input = new Scanner(System.in);
+        String task = "";
+        while(true) {
+            task = input.nextLine();
+            System.out.println("    ____________________________________");
+            if(task.equals("bye")){
+                break;
+            } else if(task.equals("list")) {
+                listTasks();
+            } else if(task.contains("mark ")&& !task.contains("unmark ")) {
+                markTask(task);
+            } else if(task.contains("unmark ")) {
+                unmarkTask(task);
+            } else {
+                addTask(task);
+            }
+            System.out.println("    ____________________________________");
+        }
+    }
+
+    private static void greetUser(String greeting) {
+        if(greeting.equals("hi")) {
+            String logo = " ____        _\n"
+                    + "|  _ \\ _   _| | _____\n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+            System.out.println(logo);
+            System.out.println("    ____________________________________");
+            System.out.println("    Hello! I'm Duke");
+            System.out.println("    What can I do for you?\n");
+        }
+        else {
+            System.out.println("    ____________________________________");
+            System.out.println("    Bye. Hope to see you again soon!");
+            System.out.println("    ____________________________________");
+        }
+    }
+
+    public static void listTasks() {
+        System.out.println("    Here are the tasks in your list:");
+        for(int i=0;i< taskList.length;i++) {
```

Comment:
Maybe a space between the semicolons in the for(...) can read readability?
for (initialization; condition; update) {
    statements;
}